src/pages/docs/src/codegen/paths.js

```diff
@@ -109,6 +109,18 @@ function getTools() {
   return getDocsFrom(tools);
 }
 
+function getBundlers() {
+
+  const toolingRoot = resolve(packagesRoot, 'tooling');
+  const webpackRoot = resolve(resolve(toolingRoot, 'piral-cli-webpack'));
+  const parcelRoot = resolve(resolve(toolingRoot, 'piral-cli-parcel'));
```

Comment:
Why `resolve(resolve(...))`? JUst a single `resolve` should it.

Reply:
ups :stuck_out_tongue_winking_eye: , changed to use just one resolve